csharp/benchmark/SeedLang.Benchmark/Program.cs

```diff
@@ -12,13 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System;
+using System.Collections.Generic;
 using BenchmarkDotNet.Running;
+using CommandLine;
+using CommandLine.Text;
 
 namespace SeedLang.Benchmark {
   class Program {
-    static void Main(string[] _) {
-      BenchmarkRunner.Run<FibBenchmark>();
-      BenchmarkRunner.Run<SumBenchmark>();
+    internal enum BenchmarkType {

```

Comment:
BenchmarkDotNet supports several console arguments. Is it possible to use the `--filter` to implement this feature?

Please refer: https://benchmarkdotnet.org/articles/guides/console-args.html

Reply:
Good! Let's switch to the solution of BenchmarkSwitcher.